### DISCUSSION TITLE
I want accessibility tools front & center

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#194] [Wants]
Accessibility tools should be center stage, not buried. Most Dev Tools are good and getting better at evaluating and presenting programmatically detectable issues. But that is a process that is manually initiated in a near-hidden tab. The idea of “inspect element” should not be limited to the declared style and compiled output. It should first indicate an accessibility status.

> Dear human inspecting this element, before you try to unveil what CSS features and properties have been applied, please note this fails {x.x.x} criterion of WCAG, and requires further inspection of {consideration}.

Accessibility needs more visibility (pun intended). Dev Tools can help.

<hr>

https://webwewant.fyi/wants/12/

**Comment by argyleink** [comment]
I agree Devtools can help, and there should be more! Let's continue pushing this 💯 

meanwhile, [VisBug](https://a.nerdy.dev/gimme-visbug) can stop gap many common inspection cases with it's Accessibility Inspector. On hover it'll grab aria roles, id's and names, titles, and of course give you a contrast score for the colors it can crawl. Maybe it can help you in the interim while we get more robust tooling?

<details>
    <summary>Failing contrast found on this page with  VisBug</summary>

   <img src="https://user-images.githubusercontent.com/1134620/126505052-a5c3ec4a-97ff-47df-b016-151a37fd2291.png">

</details>

<details>
    <summary>Inspecting an interactive element</summary>

    
<img src="https://user-images.githubusercontent.com/1134620/126505361-a54ed9e5-9419-4f64-80fa-f9e8bdd6f0b0.png">


</details>
